modular_skyrat/modules/mapping/voidraptor/code/mob.dm

```diff
@@ -0,0 +1,98 @@
+/mob/living/simple_animal/hostile/retaliate/tegu
```

Comment:
Don't we already have a file for modular pets like that one Botany bee?

Reply:
Was told to put the code out of master files, so here it goes, unless I should return it.